src/shogun/features/StringFeatures.cpp

```diff
@@ -2081,6 +2086,32 @@ bool CStringFeatures<ST>::obtain_from_char_features(CStringFeatures<CT>* sf, int
 	return true;
 }
 
+template<class ST>
+CStringFeatures<ST>* CStringFeatures<ST>::clone() const
+{
+	return new CStringFeatures<ST>(*this);
```

Comment:
note that `clone` and `equals` are implemented in `SGObject`, so no need to override them here, unless there are problems with the parameter framework (in which case it would be better to solve them)

Reply:
There where no problems so I removed the ones I added